stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt

```diff
@@ -242,8 +252,7 @@ class ManageStockViewModel @Inject constructor(
 
         _screenState.postValue(
             TableScreenState(
-                tables = tables,
-                overwrittenRowHeaderWidth = 200F
```

Comment:
The Stock use case has the particularity of start with the row header width as 200F taking almost the half of the screen. Did you take this in account for the reset and the original width?

Reply:
Yes, in fact this overwrittenRowHeaderWidth was nos been used.